src/Xamarin.Android.Build.Tasks/Properties/Resources.Designer.cs

```diff
@@ -437,7 +437,9 @@ internal class Resources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to &apos;{0}&apos; is using a deprecated debug information level. Set the debugging information to Portable in the Visual Studio project property pages or edit the project file in a text editor and set the &apos;DebugType&apos; MSBuild property to &apos;portable&apos; to use the newer, cross-platform debug information level. If this file comes from a NuGet package, update to a newer version of the NuGet package or notify the library author..
+        ///   Looks up a localized string similar to &apos;{0}&apos; is using a deprecated debug information level.
+        ///Set the debugging information to Portable in the Visual Studio project property pages or edit the project file in a text editor and set the &apos;DebugType&apos; MSBuild property to &apos;portable&apos; to use the newer, cross-platform debug information level.
+        ///If this file comes from a NuGet package, update to a newer version of the NuGet package or notify the library author..
```

Comment:
Should we remove the extra `.`?

Reply:
Nope, that's how the Visual Studio _.resx_ editor generates the comment, so that can stay. Good question though!